cache/README.md

```diff
@@ -0,0 +1,67 @@
+# Cache
+
+## Cache object
+
+```json
+{
+  "type": "string",
+  "timestamp": "object"
+}
+```
```

Comment:
What is this?

Reply:
Not needed anymore, changed the whole structure of the API